nautobot/extras/views.py

```diff
@@ -1463,7 +1463,7 @@ class JobResultListView(generic.ObjectListView):
     List JobResults
     """
 
-    queryset = JobResult.objects.select_related("job_model", "user").prefetch_related("logs")
+    queryset = JobResult.objects.defer("result").select_related("job_model", "user").prefetch_related("logs")
```

Comment:
This and the change below are different than the changes made in 7ffed896ec5ca4f61d0c23a637306192f34df2fd, might be worth looking at very closely to see if they're causing the CI failures.